survey-runner.tf

```diff
@@ -66,6 +66,7 @@ module "survey-runner-on-ecs" {
   aws_secret_key                 = "${var.aws_secret_key}"
   dns_zone_name                  = "${var.dns_zone_name}"
   ecs_cluster_name               = "${module.eq-ecs.ecs_cluster_name}"
+  aws_alb_arn                    = "${module.eq-ecs.aws_alb_arn}"
```

Comment:
This has also been added to line 85 so in twice now for the same module.

Reply:
I have fixed this now